algorithms/ar-bsrt/csharp/BubbleSort.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace BubbleSort
+{
+    public class BubbleSort
+    {
+        public static int[] GetBubbleSortedArray(int[] array)
+        {
+            int temp;
+            for (int i = 0; i < array.Length; i++)
+            {
+                for (int j = 0; j < array.Length - 1; j++)
+                {
+                    if (array[j] > array[j + 1])
+                    {
+                        temp = array[j + 1];
+                        array[j + 1] = array[j];
+                        array[j] = temp;
+                    }
+                }
+            }
+
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new int[] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
+
+            Console.WriteLine("Initial array:");
+            foreach (var element in array)

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)